packageV6.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@percy/storybook",
-  "version": "4.3.7",
+  "version": "5.0.0-beta.1",
```

Comment:
`"private": true` make this private so we won't publish this file, and we can remove version information from here.